### DISCUSSION TITLE
'nuxi generate' does not fail on errors

### DISCUSSION THREAD

**Discussion by uuf6429** [nuxt/nuxt#17332] [Questions]
This question also popped up previously in nuxt 2 and was address by adding a `--fail-on-error` CLI argument to `nuxt generate`: https://github.com/nuxt/nuxt.js/issues/4991

At some point, this functionality seems to have somehow been ported to nuxi or bridge, judging by #3134.
However, that file ([packages/bridge/src/nitro.ts](https://github.com/nuxt/framework/pull/3134/files#diff-9691911220c393199e775f8762a85841d12492fbd90d2e81d4480e9f982b8c34)) seems to have been (re)moved in later versions of the code.
A [github search in nuxt/framework](https://github.com/nuxt/framework/search?q=%22fail-on-error%22) also didn't result in anything relevant (except that issue/pr).

Am I missing something?

**Comment by danielroe** [comment]
For bridge, see https://github.com/nuxt/bridge.
